ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -295,8 +301,11 @@ initNetwork registry nodeArgs kernel RunNetworkArgs{..} = do
 
     -- serve downstream nodes
     connTable  <- newConnectionTable
+    peerStatesVar <- newPeerStatesVar
+    -- every 200s clean peer states
+    forkLinkedThread registry (NodeToNode.cleanPeerStates 200 peerStatesVar)
```

Comment:
Don't ignore the returned thread (a warning currently, causing CI to fail) but add it to the list of `threads` at the end of the function.